src/components/ae-fraction/ae-fraction.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <span class="ae-fraction" :class="[ fill ]">
+    <sup>{{ numerator }}</sup>
+    &frasl;
+    <sub>{{ denominator }}</sub>
+  </span>
+</template>
+<script>
+export default {
+  name: 'ae-fraction',
+  props: {
+    /**
+     * Fill property changes the color state of the fraction component.
+     * Select something between: `primary, secondary, neutral, alternative, white, black`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+        'white',
+        'black',
+      ].includes(value),
+    },
+
+    /**
+     * The numerator property defines
+     * the upper number in the fraction.
+     */
+    numerator: {
+      type: [Number, String],
+      default: 0,
+    },
+
+    /**
+     * The numerator property defines
+     * the lower number in the fraction.
+     */
+    denominator: {
+      type: [Number, String],
+      default: 0,
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+@import '../../styles/variables/colors';
+@import '../../styles/variables/typography';
+
+.ae-fraction {
+  display: inline-block;
+  font-family: $font-sans;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 0;
+  word-spacing: -4px;
```

Comment:
Shouldn't word spacing be relative to the font size?

Reply:
explicitly added front-size.

Reply:
I think you should set this value in `em` and remove explicit font-size.

Reply:
Fixed, not `em` but `rem`

Reply:
Okay, but if somebody changes font-size of this element he has to adjust `word-spacing` because now it is relative to the global font size. If word-spacing be in em, then it will correspond to current font-size of this element, so probably no need to adjust word-spacing. It looks more reasonable, why don't use `em` in this case?

Reply:
This element does not need to have a dynamic word spacing! It does not need to be dependent on a parent element. So in this case `em` does not make sense.